src/firefox/index.js

```diff
@@ -109,6 +109,11 @@ export type FirefoxRunnerFn =
   (params: FirefoxRunnerParams) => Promise<FirefoxRunnerResults>;
 
 
+export type FirefoxRunResponse = {|
```

Comment:
We should call this `FirefoxInfo` to make it more specific to the data rather than the method that returns the data.